src/styles/components/_cards.scss

```diff
@@ -0,0 +1,91 @@
+.cards {
+  padding: 50px 0;
+  display: flex;
+  flex-wrap: wrap;
+
+  &__card {
+    border: 1px solid $color-frames;
+    border-radius: 5px;
+    width: 166px;
+    padding: 32px 16px 16px;
+    display: flex;
+    flex-direction: column;
+    color: $color-main;
+    align-items: center;
+    margin-bottom: 46px;
+    margin-right: 48px;
+  }
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__product-info {
+    margin-bottom: 16px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    margin-bottom: 4px;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    color: $color-secondary;
+    font-size: 10px;
+  }
+
+  &__review {
+    display: flex;
+    width: 100%;
+    margin-bottom: 24px;
+    justify-content: space-between;
+    align-items: center;
+
+    &-value {
+      font-size: 10px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-label {
+      font-size: 12px;
+      color: $color-secondary;
+    }
+
+    &-value {
+      font-weight: 700;
+      font-size: 16px;
+    }
+  }
+
+  &__button-link {
+    width: 100%;
+  }
+  &__button {
+    background-color: $color-blue;
+    width: 100%;
+    padding: 11px 0;
+    border-radius: 5px;
+    border-style: none;
+    border: 1px solid rgba(0, 0, 0, 0);
+    color: white;
+    font-weight: 700;
+    font-size: 14px;
+
+    &:hover {
+      background-color: #fff;
+      border: 1px solid $color-blue;
```

Comment:
you can just change the `border-color` here, you dont need to redeclare the entire border